apps/prairielearn/src/pages/instructorStudents/components/StudentsTable.tsx

```diff
@@ -0,0 +1,193 @@
+import { type SortDirection, type Table, flexRender } from '@tanstack/react-table';
+import { notUndefined, useVirtualizer } from '@tanstack/react-virtual';
+import { type JSX, useRef } from 'preact/compat';
+
+import type { StudentRow } from '../instructorStudents.shared.js';
+
+function SortIcon({ sortMethod }: { sortMethod: null | SortDirection }) {
+  if (sortMethod === 'asc') {
+    return <i className="bi bi-sort-up-alt"></i>;
+  } else if (sortMethod === 'desc') {
+    return <i className="bi bi-sort-down"></i>;
+  } else {
+    return <i className="bi bi-arrows-expand opacity-75 text-muted"></i>;
```

Comment:
Consider using https://icons.getbootstrap.com/icons/arrow-down-up/ here. I know you mentioned that FontAwesome might have a better icon, but we'd like to use Bootstrap Icons if at all possible.

Reply:
Done in [ca19f54](https://github.com/PrairieLearn/PrairieLearn/pull/12211/commits/ca19f548a4011ad4809dd5090b627d743a04b06e).